src/runtime/parallel/coordinator.ts

```diff
@@ -141,6 +160,24 @@ export default class Coordinator {
     }
   }
 
+  awakenWorkers(): void {
+    const workers = _.values(this.workers)
+    _.each(workers, (worker): boolean => {
```

Comment:
I think you can avoid using `_.values` and just pass `this.workers` directly into `_.each`